MachoMaheen_zkthon

MachoMaheen opened the pull request:
## Description

This pull request is for the zkEVM Challenges 1, 2, and 3.

## Checklist
 
 - [x] I have created a new file named solution-1.md for the challenge 1.
 - [x] I have created a new file named solution-2.md for the challenge 2.
 - [x]  I have created a new file named solution-3.md for the challenge 3.
 - [x] I have added the details of my submission to the [Airtable form](https://airtable.com/shr21z0FfPImZfYBQ).
 - [x] I have named my branch MachoMaheen_zkThon.

MachoMaheen committed:
added solution-1.md/completed challenge 1

MachoMaheen committed:
Added solution.md -- Completed 2nd Challenge

MachoMaheen committed:
Added solution3.md -- Completed 3rd Challenge

MachoMaheen committed:
file name changed from solutions-2.md to solution-2.md

PriyathamVarma reviewed (changes requested):
Thank you for the participation ser. But as part of challenge-3 we expect you to paste the code you used to interact with the smart contracts. This could be in your preferred language.

MachoMaheen committed:
Update solution-3.md

MachoMaheen commented:
I updated solution-3.md file. Can you check it. Also in challenge-3.md file, it is clearly specified that the participants can either use ethersjs or web3js to interact with the deployed smartcontract or we can use the direct method call(manually using the contract)of the contract deployed using the remix. The challenge-3.md clearly says that both of the ways will be accepted for the challenge. Anyway i added the ethersjs code to interact the smartcontract in the solution-3.md file along with transaction hash and contract address.
Please verify

MachoMaheen committed:
Update solution-3.md

PriyathamVarma reviewed (approved):
Super job ser.

megabyte0x reviewed (approved):
Great work
